packages/graphql_server/src/service/twitter-scraping-service.ts

```diff
@@ -0,0 +1,165 @@
+import axios, { AxiosRequestConfig } from 'axios'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+type TwitterUser = {
```

Comment:
please refactor the types into the types folder under the name twitterScrapingService.types.ts

Reply:
d.ts